src/renderer/Generics/redesign/LogMessage/logs.css.ts

```diff
@@ -5,10 +5,11 @@ export const container = style({
   position: 'relative',
   maxWidth: '100%',
   width: 'inherit',
-  height: '100%',
+  height: 'auto',
   display: 'flex',
   flexDirection: 'column',
   flex: 1,
+  paddingTop: '32px', // needed on mac so close logs btn isnt under drag bar
```

Comment:
I have this issue fixed in my fix/various branch - seems like this issue came with the newest electron version, so we can remove this padding

Reply:
actually, we should merge with this, will merge main into fix/various and take this padding off then.